src/assets/i18n/en.json

```diff
@@ -253,6 +253,7 @@
     "add-cafe": "Add a cafe",
     "admin": "Administrator",
     "my-habits": "My habits",
+    "ubs": "UBS courier",
```

Comment:
we have the same text in 4 places in 1 file, how about having only 1 ?

Reply:
unnecessary translations were removed with this commit 17b2ed1a78d91e4e158a46199ff72ff2f89963c1